apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-examples/platform-file-uploader-compact-example.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import { FileUploaderSelectionChangeEvent } from '@fundamental-ngx/platform';
+
+@Component({
+    selector: 'fdp-platform-file-uploader-compact-example',
+    templateUrl: './platform-file-uploader-compact-example.component.html'
```

Comment:
add OnPush changedetection strategy to all examples

Reply:
not necessary, to have it in all the examples.